### PR TITLE
Use findtools to install find

### PR DESCRIPTION
**Pull request by pvanheus**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by pvanheus** [comment]
BUSCO uses `find` internally and relies on the `-delete` flag that is not present in the BusyBox version of `find`. This recipe adds a dependency on GNU findtools to ensure that an appropriate `find` is present for BUSCO to use.

File: recipes/busco/meta.yaml [hunk 1]
```diff
@@ -29,10 +30,10 @@ test:
   requires:
     - wget
   commands:
-    - run_BUSCO.py -h
-    - run_busco -h
-    - generate_plot.py -h
-    - generate_plot -h
+    - run_BUSCO.py -h >/dev/null
```
Comment (bgruening) [inline recipes/busco/meta.yaml]: Why is this needed @pvanheus?

File: recipes/busco/run_test.sh [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 set -euo pipefail
 
+# test that find has the -delete flag, which BUSCO uses
+mkdir findtest ; cd findtest ; touch file ; find . -delete ; cd ..
```
Comment (bgruening) [inline recipes/busco/run_test.sh]: Why can this not be in the meta.yaml?

File: recipes/busco/meta.yaml (hunk 1 above)
Comment (pvanheus) [reply recipes/busco/meta.yaml]: It makes for less noisy test output - not needed, but it makes spotting the source of errors easier.

File: recipes/busco/run_test.sh (hunk 2 above)
Comment (pvanheus) [reply recipes/busco/run_test.sh]: This whole shell command? I originally had it in meta.yaml, but this is rather verbose and thus since there was already a run_test.sh, I moved it there.

Comment (bgruening) [reply recipes/busco/run_test.sh]: The current Docker tests, do not use external run_test files. So all test that you also want to run in Docker should be in meta.yaml.

File: recipes/busco/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/busco/meta.yaml]: But if the error message is "missing foobar" that is printed to stdout you are not seeing it, right?
Your call. I find more output good in tests.

**Comment by bgruening** [comment]
Thanks!
